entity-framework/core/get-started/netcore/new-db-sqlite.md

```diff
@@ -1,100 +1,91 @@
+# .NET Core - New database
 
-> [!IMPORTANT]
-> The [.NET Core SDK](https://www.microsoft.com/net/download/core) 1.0.0 no longer supports `project.json` or Visual Studio 2015. Everyone doing .NET Core development is encouraged to [migrate from project.json to csproj](https://docs.microsoft.com/dotnet/articles/core/migration/) and [Visual Studio 2017](https://www.visualstudio.com/downloads/).
+In this walkthrough, you will create a .NET Core console app that performs basic data access against a SQLite database using Entity Framework. You will use migrations to create the database from your model. See [ASP.NET Core - New database](xref:core/get-started/aspnetcore/new-db) for a Visual Studio version using ASP.NET Core MVC.
 
-In this walkthrough, you will build a .NET Core console application that performs basic data access against a SQLite database using Entity Framework. You will use migrations to create the database from your model.
-
-> [!TIP]
-> You can view this article's [sample](https://github.com/aspnet/EntityFramework.Docs/tree/master/samples/core/GetStarted/NetCore/ConsoleApp.SQLite) on GitHub.
+Notes: 
+- This documentation is for EF Core. For EF6.x, see [Entity Framework 6](../../../ef6/index.md).
+- The [.NET Core SDK](https://www.microsoft.com/net/download/core) 1.1.x no longer supports `project.json` or Visual Studio 2015. We recommend you [migrate from project.json to csproj](https://docs.microsoft.com/dotnet/articles/core/migration/). If you are using Visual Studio, we recommend you migrate to [Visual Studio 2017](https://www.visualstudio.com/downloads/).
+- You can view this article's [sample](https://github.com/aspnet/EntityFramework.Docs/tree/master/samples/core/GetStarted/NetCore/ConsoleApp.SQLite) on GitHub.
 
 ## Prerequisites
 
 The following prerequisites are needed to complete this walkthrough:
 * An operating system that supports .NET Core.
-* [The .NET Core SDK](https://www.microsoft.com/net/core) provides the `dotnet` command line tool, which will be used to build and run your application.
-* A text editor or IDE of your choice.
+* [The .NET Core SDK](https://www.microsoft.com/net/core).
 
 ## Create a new project
 
-* Create a new `ConsoleApp.SQLite` folder for your project and use the `dotnet` command to populate it with a .NET Core application.
+* Create a new `ConsoleApp.SQLite` folder for your project and use the `dotnet` command to populate it with a .NET Core app.
 
 ```bash
 mkdir ConsoleApp.SQLite
 cd ConsoleApp.SQLite/
-dotnet new
+dotnet new console
 ```
 
 ## Install Entity Framework
 
 To use EF Core, install the package for the database provider(s) you want to target. This walkthrough uses SQLite. For a list of available providers see [Database Providers](../../providers/index.md).
 
-*  Modify the `ConsoleApp.SQLite.csproj` file so that it matches the following.
+*  Replace the `ConsoleApp.SQLite.csproj` contents with the following:
```

Comment:
Would help to know what is being changed and why here -- can you highlight changed lines and add some explanatory text?